File: Solutions/2794/java/program.java
// Copy from Quera

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner=new Scanner(System.in);
        int x1=scanner.nextInt(),y1=scanner.nextInt();
        int x2=scanner.nextInt(),y2=scanner.nextInt();
        int x3=scanner.nextInt(),y3=scanner.nextInt();
        int x4,y4;
        if(x1==x2)x4=x3;
        else if(x1==x3)x4=x2;
        else x4=x1;
        if(y1==y2)y4=y3;
        else if(y1==y3)y4=y2;
        else y4=y1;
        System.out.println(x4+" "+y4);
    }
}

File: Solutions/2794/c++/program.cpp
// Copy from Quera

// In the name of God

#include <iostream>
using namespace std;

int main()
{
    int a1, b1, a2, b2, a3, b3;
    cin >> a1 >> b1 >> a2 >> b2 >> a3 >> b3;
    if (a1 == a2)
        cout << a3;
    else if (a1 == a3)
        cout << a2;
    else
        cout << a1;

    cout << ' ';

    if (b1 == b2)
        cout << b3;
    else if (b1 == b3)
        cout << b2;
    else
        cout << b1;

    cout << '\n';
    return 0;
}
